lib/matplotlib/tests/test_text.py

```diff
@@ -707,9 +708,14 @@ def test_large_subscript_title():
      (0.3, 0, 'right'),
      (0.3, 185, 'left')])
 def test_wrap(x, rotation, halign):
-    fig = plt.figure(figsize=(6, 6))
+    fig = plt.figure(figsize=(18, 18))
+    gs = GridSpec(nrows=3, ncols=3, figure=fig,
+                  left=0, right=1, bottom=0, top=1)
```

Comment:
I think these left/right/bottom/top settings are redundant since subfigures ignore the gridspec white space settings.  I just tried the test without them and it still passed.

That said, these settings clearly don't do any harm either, so I leave it up to you if you want to do anything about it.  Please self-merge at will.